src/components/LatestArticles.jsx

```diff
@@ -0,0 +1,32 @@
+export const LatestArticles = ({ articles }) => {
+  const latestArticles = articles
+    .toSorted((a, b) => a.created_at - b.created_at)
+    .slice(1, 5);
```

Comment:
This is not the ideal way to do this as we should be using our api to simply give us the articles in the correct order (maybe using queries), but if you didnt get to that point in the core tasks for the BE project then this is fine for now and can be replaced at a later time when that functionality is added.